syntax = "proto3";
package grpc;
message Empty {}
// I called it Note because message Message sounds complicated
message Note {
    string name = 1;
    string message = 2;
}
service PubSubServer{
    // This bi-directional stream makes it possible to send and receive Notes between 2 persons
    rpc PubSubStream (Empty) returns (stream Note);
    rpc SendNote (Note) returns (Empty);
}